docs/docs/articles/run-tests-with-github-actions.md

```diff
@@ -1,4 +1,5 @@
 # Run Tests with GitHub Actions
+**If you need more control over your flow or to access a private cluster, use Testkube Action instead.**
```

Comment:
It should link to the documentation of `Testkube Action`

Reply:
Shouldn't it actually point to docs.testkube.io documentation, instead of GH marketplace?